### PR TITLE
Added SELECT2_THEME settings

### PR DESCRIPTION
**Pull request by CleitonDeLima**
Resolve: #101

### CODE REVIEW

File: django_select2/conf.py [hunk 1]
```diff
@@ -87,6 +87,14 @@ class Select2Conf(AppConf):
 
         SELECT2_CSS = 'assets/css/select2.css'
 
+    If you want to add more css (usually used in select2 themes), add a line
+    in settings.py like this::
+
+        SELECT2_CSS = (
+            'assets/css/select2.css',
+            'assets/css/select2-theme.css',
+        )
```
Comment (codingjoe) [inline django_select2/conf.py:96]: Arrays or arbitrary length should be a list, according to Guido, Django's docs also removed the usage of tuples, so I guess we should do the same here.

**Comment by codingjoe** [comment]
Pretty much perfect, thanks @CleitonDeLima, very impressive.

File: django_select2/conf.py (hunk 1 above)
Comment (CleitonDeLima) [reply django_select2/conf.py:96]: Perfect, i'll change the media property too.

**Review by codingjoe — APPROVED** [review]
THX 👍

**Comment by GorlikItsMe** [comment]
Hi. Can someone make an example of how to change the theme in documentation?

**Comment by codingjoe** [comment]
Excellent idea. @CleitonDeLima would you do the honors?

**Comment by CleitonDeLima** [comment]
Hi @GorlikItsMe 

@codingjoe , documentation is not generated through docstrings?  
I remember adding some comments about it.

**Comment by GorlikItsMe** [comment]
https://django-select2.readthedocs.io/en/latest/django_select2.html#django_select2.conf.Select2Conf.CSS

SELECT2_CSS AND SELECT2_JS are
But not SELECT2_THEME

**Comment by CleitonDeLima** [comment]
> SELECT2_CSS AND SELECT2_JS are But not SELECT2_THEME

see [here](https://github.com/codingjoe/django-select2/pull/103/files#diff-ca927d3bfbe96c014cfe2ab3afbaa414a9357925b015a2122903b1c8523dab1a)  
I don't think the documentation has been updated yet.

**Pull request merged**